lib/BridgeAbstract.php

```diff
@@ -182,6 +182,19 @@ protected function setInputs(array $inputs, $queriedContext){
 		}
 	}
 
+	/**
+	 * Get bridge specific configuration value for the given key
+	 *
+	 * Each bridge has its own section in the central configuration file
+	 * using the same name as its class name.
+	 *
+	 * @param string $key Given key
+	 * @return string Requested value
+	 */
+	protected function getConfig($key) {
```

Comment:
This should be merged soon. One small suggestion to emphasize the intended use (similar to #1380):

```suggestion
	protected function fetchConfigValue($key) {
```

`get` can be confused with public getters.